package-res/resources/web/pentaho/data/filter/IsEqual.js

```diff
@@ -0,0 +1,78 @@
+/*!
+ * Copyright 2010 - 2016 Pentaho Corporation.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define([
+  "./AbstractPropertyFilter"
+], function(AbstractPropertyFilter) {
+  "use strict";
+
+  /**
+   * @name IsEqual
+   * @memberOf pentaho.data.filter
+   * @class
+   * @extends pentaho.data.filter.AbstractPropertyFilter
+   * @amd pentaho/data/filter/IsEqual
+   *
+   * @classdesc The `IsEqual` class implements a type of AbstractPropertyFilter {@link pentaho.data.filter.AbstractPropertyFilter}.
```

Comment:
@classdesc A filter that defines the set of items having the value of a property equal to a certain value.